jsonnet/components/node-exporter.libsonnet

```diff
@@ -191,6 +191,8 @@ function(params)
                           memory: '32Mi',
                         },
                       },
+                      // TODO(slashpai) node-exporter has issue in rolling out overriding till issue fixed
```

Comment:
Can you add more context? It's not clear to me what the issue is and where/how it is fixed.

Reply:
For the same issue found in logs https://github.com/openshift/cluster-monitoring-operator/pull/1556#issuecomment-1040562199. I was trying to see if that was causing issue so tried overriding the [change](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/node-exporter.libsonnet#L185-L189)